src/types.jl

```diff
@@ -28,13 +28,17 @@ struct Logger
     verbosity::Int
     experiment_name::String
     artifact_location::Union{String,Nothing}
+    _logging_channel::Channel{Tuple}
```

Comment:
Nit picking point: Can't  `_logging_channel` be `logging_channel` ?

Reply:
I considered `_logging_channel` instead of `logging_channel` because I want it to be like a "private" field. The user is not intended to touch that field. However, I discovered that I made a mistake on `open_logging_channel()` because I'm returning a value instead of setting it up in the logger.

Reply:
I think in Julia fields are generally considered private, especially if they are not referenced in any doc-string. To make them public you would ordinarily provide an accessor function. But there's no harm in this - I just found it a bit strange.